src/modules/Transcript/EventDetails/index.tsx

```diff
@@ -16,6 +19,30 @@ interface EventDetailsUIProps extends EventDetailsSharedProps {}
 
 export function EventDetailsUI(props: EventDetailsUIProps): ReactElement {
     const { event, eventDetailsExpanded, toggleEventDetails } = props;
+    const [transcriptProxy, setTtranscriptProxy] = useState<string | null>(null);
+    const userQuery = useQuery<CurrentUserQuery>({
+        requestPolicy: 'cache-only',
+        query: gql`
+            query CurrentUserQuery {
+                currentUser {
+                    id
+                    apiKey
+                }
+            }
+        `,
+    });
+
+    useEffect(() => {
+        console.log(process.env.NODE_ENV);
+        if (!transcriptProxy && userQuery.state.data?.currentUser?.apiKey) {
+            setTtranscriptProxy(
+                `https://audio` +
+                    (process.env.NODE_ENV !== 'production' ? `-dev` : '') +
+                    `.aiera.com/api/events/${event.id}/audio/transcript?api_key=${userQuery.state.data.currentUser.apiKey}`
+            );
+        }
+    }, [transcriptProxy, userQuery]);
```

Comment:
this will get called more than once.. which is fine.. but I am wondering if it is worth keeping in state... vs just using userQuery.state.data?.currentUser?.apiKey as the conditional on line 122?